cypress/integration/composite_level_monitor_spec.js

```diff
@@ -82,40 +82,15 @@ describe('CompositeLevelMonitor', () => {
 
       // Wait for monitor to be created
       cy.wait('@createMonitorRequest').then((interceptor) => {
-        const monitorId = interceptor.response.body.resp._id;
+        // Verify the monitor name on details page
+        cy.contains(SAMPLE_VISUAL_EDITOR_MONITOR);
 
-        cy.contains('Loading monitors');
-        cy.wait('@getMonitorsRequest').then((interceptor) => {
-          const monitors = interceptor.response.body.monitors;
-          const monitor1 = monitors.filter((monitor) => monitor.name === 'monitor_1');
-          const monitor2 = monitors.filter((monitor) => monitor.name === 'monitor_2');
-
-          // Let monitor's table render the rows before querying
-          cy.wait(1000).then(() => {
-            cy.get('table tbody td').contains(SAMPLE_VISUAL_EDITOR_MONITOR);
-
-            // Load sample data
-            cy.insertDocumentToIndex(
-              sample_index_1,
-              undefined,
-              sampleCompositeJson.sample_composite_associated_index_document
-            );
-            cy.insertDocumentToIndex(
-              sample_index_2,
-              undefined,
-              sampleCompositeJson.sample_composite_associated_index_document
-            );
+        const monitorId = interceptor.response.body.resp._id;
```

Comment:
Does this get used anywhere? It doesn't look like it.

Reply:
Fixed